CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 1.5.3 (August 30, 2024)
+
+IMPROVEMENTS:
+
+* Update `Update `github.com/hashicorp/consul/proto-public` to v0.6.2. [[GH-585](https://github.com/hashicorp/consul-dataplane/pull/585)]
+* Update multiple dependencies. [[GH-602](https://github.com/hashicorp/consul-dataplane/pull/602)]
+* Update ubi base image to `ubi9-minimal:9.4`. [[GH-602](https://github.com/hashicorp/consul-dataplane/pull/602)]
+
+
```

Comment:
Notice that the changelogs here are since 1.5.1 and exclude 1.5.2. The plan is to completely redact/remove 1.5.2 to avoid it being used.

Reply:
Should we maybe include an entry for that version (or a line in 1.5.3 at the top), stating that 1.5.2 is intentionally skipped due to a vulnerability? We've had missing entries in the past due to human error, so it might help to clarify.

Reply:
@zalimeni added an entry for 1.5.2